app.rb

```diff
@@ -0,0 +1,149 @@
+require './person'
+require './book'
+require './classroom'
+require './rental'
+require './student'
+require './teacher'
+require 'pry'
+class App
+  def run
+    @books = []
+    @person = []
+    @rental = []
+    puts 'Welcome to School Library App!'
+    puts "\n"
+    options
+  end
+
+  def options
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all retnals for a given person id'
+    puts '7 - Exit'
+    num = gets.chomp.to_i
+    perform_switch(num)
+    options
+  end
+
+  # rubocop:disable Metrics/CyclomaticComplexity
+  def perform_switch(choice)
+    case choice
+    when 1
+      list_books
+    when 2
+      list_people
+    when 3
+      create_person
+    when 4
+      create_book
+    when 5
+      create_rental
+    when 6
+      list_rentals
+    when 7
+      exit_from_menu
+    else puts 'Only digits 1 to 7 are allowed'
+    end
+  end
+  # rubocop:enable Metrics/CyclomaticComplexity
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    num = gets.chomp.to_i
+    case num
+    when 1
+      @person.push(create_student)
+      puts 'Student created successfully'
+    when 2
+      @person.push(create_teacher)
+      puts 'Teacher created successfully'
+    end
+  end
+
+  def create_student
+    puts 'Age: '
+    age = gets.chomp.to_i
+    puts 'Name: '
+    name = gets.chomp.to_s
+    puts 'Has parent permission? [Y/N]: '
+    parent_permission = gets.chomp.downcase
+    case parent_permission
+    when 'y'
+      Student.new(age, name)
+    when 'n'
+      Student.new(age, name, parent_permission: false)
+    else
+      puts 'Please enter Y/N[y/n]'
+      create_student
+    end
```

Comment:
- [ ] I faced the error in the image below when I wanted to create a student.

![second-review](https://user-images.githubusercontent.com/53124548/158024932-23e11700-e8d6-445d-b846-2b7c3ae76bfc.png)

Reply:
I don't know what's happening on your side, but its working perfectly fine on my side